llm/README.md

```diff
@@ -1,3 +1,9 @@
+<br />
+<p align="center">
+   <img src="./loss-curve.png" width="100%" height="50%"/>
```

Comment:
can you fix the pixels of the width instead? I think that helps not making excessively large images.

Also, you can use Preview to create a light dark version. e.g.
```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="./dark.png">
  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="./light.png">
</picture
```